src/compiler/scala/tools/nsc/Global.scala

```diff
@@ -1058,12 +1129,6 @@ class Global(var currentSettings: Settings, reporter0: Reporter)
    */
   override def currentRunId = curRunId
 
-  def echoPhaseSummary(ph: Phase) = {
```

Comment:
what do we gain by changing this method. We need to keep the focus on the chnage, not optimisations and refactors that are orthoganal IMO

Reply:
It's related to some extend.
I changed `typer`'s `run` method to call super, previously it was re-implementing it.
`echoPhaseSummary` was one of the methods which are no longer needed when we are not re-implementing code from the run.